### PR TITLE
added navbar to profile page

### PR DESCRIPTION
**Pull request by jeromes123**
**OVERVIEW**
1. Extended the navbar to profile page

### CODE REVIEW

File: src/resources/views/profile/index.blade.php [hunk 1]
```diff
@@ -1,9 +1,11 @@
 @extends('layouts.app')
 
-@section('title', 'My Thoughts')
+@section('title', $user->username)
```
Comment (mikco-hipe) [inline src/resources/views/profile/index.blade.php]: what if $user->username is null, what will be displayed?

**Review by mikco-hipe — COMMENTED** [review]
Please check!

File: src/resources/views/profile/index.blade.php (hunk 1 above)
Comment (jeromes123) [reply src/resources/views/profile/index.blade.php]: i have implemented ternary operator to handle null value

**Review by mikco-hipe — APPROVED** [review]
LGTM 🙇

**Pull request merged**